tasks/parse-pdr/tests/parse_pdrs_test.js

```diff
@@ -291,24 +291,16 @@ test.serial('Parse a PDR without a granuleIdFilter in the config', async (t) =>
   await validateInput(t, t.context.payload.input);
   await validateConfig(t, t.context.payload.config);
 
-  let output;
-  try {
-    output = await parsePdr(t.context.payload);
+  const output = await parsePdr(t.context.payload);
 
-    await validateOutput(t, output);
+  await validateOutput(t, output);
+
+  t.deepEqual(output.pdr, t.context.payload.input.pdr);
+  t.is(output.granules.length, 2);
+  t.is(output.granulesCount, 2);
+  t.is(output.filesCount, 2);
+  t.is(output.totalSize, 3952643);
 
-    t.deepEqual(output.pdr, t.context.payload.input.pdr);
-    t.is(output.granules.length, 2);
-    t.is(output.granulesCount, 2);
-    t.is(output.filesCount, 2);
-    t.is(output.totalSize, 3952643);
-  }
-  catch (err) {
```

Comment:
Are we sure we won't need this anymore? This catch seems related to whether Localstack is up/down and not necessarily the FTP issue?

Reply:
If localstack is unavailable for some reason then that's an issue that we need to fix.